src/routes/controllers/articles.controller.js

```diff
@@ -162,10 +163,9 @@ export async function getArticle(req, res) {
         article,
       );
     }
-
     return errorResponse(res, 404, 'Article does not exist');
   } catch (error) {
-    return errorResponse(res, 500, 'An error occured', error.message);
+    return errorResponse(res, 500, 'database error', error);
```

Comment:
Let's make the error message generic, if the `error` variable contains errors from the database we need to remove it.